src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -63,8 +63,11 @@ public override BoundNode VisitConversion(BoundConversion node)
 
             bool wasInExpressionLambda = _inExpressionLambda;
             _inExpressionLambda = _inExpressionLambda || (node.ConversionKind == ConversionKind.AnonymousFunction && !wasInExpressionLambda && rewrittenType.IsExpressionTree());
+            InstrumentationState.IsSuppressed = _inExpressionLambda;
```

Comment:
>InstrumentationState.IsSuppressed = _inExpressionLambda; [](http://example.com/codeflow?start=12&length=56)

Should this change be tested?

Reply:
This PR doesn't change the behavior. I'd hope there is a test covering this.